packages/help-center/src/components/help-center-more-resources.tsx

```diff
@@ -26,22 +26,26 @@ export const HelpCenterMoreResources = () => {
 	const [ showWhatsNewDot, setShowWhatsNewDot ] = useState( false );
 	const sectionName = useSelector( getSectionName );
 
-	const { isBusinessOrEcomPlanUser, siteId, isSimpleSite } = useSelector( ( state ) => {
+	const { isBusinessOrEcomPlanUser, siteId } = useSelector( ( state ) => {
 		const purchases = getUserPurchases( state );
 		const purchaseSlugs = purchases && purchases.map( ( purchase ) => purchase.productSlug );
 		const siteId = getSelectedSiteId( state );
-		const site = getSite( state, siteId );
-
 		return {
 			isBusinessOrEcomPlanUser: !! (
 				purchaseSlugs &&
 				( purchaseSlugs.some( isWpComBusinessPlan ) || purchaseSlugs.some( isWpComEcommercePlan ) )
 			),
-			isSimpleSite: site && ! site.is_wpcom_atomic,
 			siteId: siteId,
 		};
 	} );
-	const { data, isLoading, setHasSeenWhatsNewModal } = useHasSeenWhatsNewModalQuery( siteId );
+
+	const isSimpleSite = useSelector( ( state ) => getIsSimpleSite( state ) ) || false;
```

Comment:
Same as before! Moreover, this could be moved inside the other `useSelector`!

Reply:
Thank you for reviewing @escapemanuele , I have applied your good suggestions. 🙇‍♂️